src/components/NavigationDrawer.vue

```diff
@@ -57,25 +67,25 @@ export default {
     pageLinks() {
       const rules = [
         {
-          text: 'Rules',
+          text: this.$t('global.rules'),
```

Comment:
Once we have them all organized, we can add inline to a component this way which is really nice.